libs/ui/editor/src/lib/panel/teaserListConfigPanel.tsx

```diff
@@ -0,0 +1,164 @@
+import {TagType, TeaserType} from '@wepublish/editor/api'
+import {useEffect, useRef, useState} from 'react'
+import {useTranslation} from 'react-i18next'
+import {Button, Drawer, Form, Schema, SelectPicker, TagPicker} from 'rsuite'
+
+import {SelectTags} from '../atoms/tag/selectTags'
+import {TeaserListBlockValue} from '../blocks/types'
+import styled from '@emotion/styled'
+import {css} from '@emotion/react'
+
+const DrawerBody = styled(Drawer.Body)`
+  padding: 24px;
+`
+
+const inputStyles = css`
+  width: 200px !important;
+`
+
+export type TeaserListConfigPanelProps = {
+  value: TeaserListBlockValue
+  onClose(): void
+  onSelect(newValue: TeaserListBlockValue): void
+}
+
+export const useTagTypeText = () => {
```

Comment:
Eh this should be useTeaserTypeText, too much tag handling in this file I guess :D